### ISSUE TITLE
Map to, or reuse, the GeoCore Ontology

### ISSUE THREAD

**Issue by brandonnodnarb** [ESIPFed/sweet#184]
as described in https://www.sciencedirect.com/science/article/pii/S0098300419306284

repo: https://github.com/BDI-UFRGS/GeoCoreOntology

I have only had a preliminary look but this appears to be an amalgamation of their previous work(s), but now available in OWL (instead of KIF, or unavailable).

**Comment by smrgeoinfo** [comment]
The ontology is constructed in a BFO framework. Mostly it names some high level classes and some disjoint constraints. 
Various projects are under way in this field... Mara Abel and MichelPerrin have been working on geoscience ontology for some time, you've probably seen Perrin and Renard 2013 'Shared Earth Modeling', if not, its good stuff.   Then there's the group at the Department of Earth Science, University of Turin, Italy and ['Ontogeneous](https://meetingorganizer.copernicus.org/EGU2019/EGU2019-13458.pdf)', which is essentially an implementation of GeoSciML v3.2 in OWL.  And the GeoSciML workgroup [activities to move into linked data](https://github.com/opengeospatial/GeoSciML/tree/master/ontology).  Some other relevant work in Sinha, GSA special paper 297, 2006-- [Richard-Geoscience Concept Models](https://pubs.geoscienceworld.org/books/book/554/chapter/3802424/Geoscience-concept-models), [Babaie et al--structural geology ontology](https://pubs.geoscienceworld.org/books/book/554/chapter/3802535/Designing-a-modular-architecture-for-the). Also... [ NADM C1 Conceptual model 2004](https://pubs.usgs.gov/of/2004/1334/). An now a new ontolgoy to suppor the LOOP3D project, should be public by the end of march....

Bottom line-- there a lots of possible reuse targets, but I'd suggest that SWEET should determine it's purpose, implement classes and properties for that, and then we can map to any number of these other models where it's useful.

**Comment by smrgeoinfo** [comment]
The [GeoscienceOntology for Loop3D](https://github.com/Loop3D/GKM) is now public and we're working on generating a gitHub release (it is the end of March :) ).

**Comment by dr-shorthair** [comment]
Snake_case class names, camelCase properties?

**Comment by smrgeoinfo** [comment]
hmmm, yes that seems to be the case....
